templates/locales/en.yml

```diff
@@ -2065,6 +2065,12 @@ en:
         interactive_not_elevated: "To be interactive, it must also be privileged."
 
       ansible:
+        ansible_host_pattern_detected: |-
+          Vagrant has detected a host range pattern in the ansible.groups option.
```

Comment:
Let's just say _... in the "groups" option._, as the `ansible` object is only a common convention.